packages/foreman/rubygem-google-cloud-compute/rubygem-google-cloud-compute.spec

```
@@ -0,0 +1,64 @@
+# template: default
+%global gem_name google-cloud-compute
+
+Name: rubygem-%{gem_name}
+Version: 0.4.0
+Release: 1%{?dist}
+Summary: API Client library for the Google Cloud Compute API
+License: Apache-2.0
+URL: https://github.com/googleapis/google-cloud-ruby
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: ruby >= 2.6
+BuildRequires: ruby >= 2.6
+BuildRequires: rubygems-devel
+BuildArch: noarch
+# end specfile generated dependencies
+
+%description
+google-cloud-compute is the official client library for the Google Cloud
+Compute API.
+
+
+%package doc
+Summary: Documentation for %{name}
+Requires: %{name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{name}.
+
+%prep
+%setup -q -n  %{gem_name}-%{version}
+
+%build
+# Create the gem as gem install only works on a gem file
+gem build ../%{gem_name}-%{version}.gemspec
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%gem_install
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%exclude %{gem_instdir}/.yardopts
+%{gem_instdir}/AUTHENTICATION.md
```

I think this is a doc file. You can probably also move it to `%files doc`:
```suggestion
%doc %{gem_instdir}/AUTHENTICATION.md
```